docs/content/latest/explore/ysql-language-features/gin.md

```diff
@@ -262,17 +260,17 @@ If write performance and storage aren't an issue, it may be worth creating a GIN
 
 ### Presplitting
 
-By default, `ybgin` indexes use a single range-partitioned tablet.
-Like regular tables and indexes, it is possible to presplit a `ybgin` index to multiple tablets at specified split points.
+By default, `gin` indexes use a single range-partitioned tablet.
+Like regular tables and indexes, it is possible to presplit a `gin` index to multiple tablets at specified split points.
 These split points are for the index, so they need to be represented in the index key format.
 This is simple for tsvector and array types, but it gets complicated for jsonb and text (`pg_trgm`).
-`jsonb_path_ops` especially should use hash partitioning since the index key is itself a hash, but hash partitioning `ybgin` indexes is currently unsupported.
+`jsonb_path_ops` especially should use hash partitioning since the index key is itself a hash, but hash partitioning `gin` indexes is currently unsupported.
```

Comment:
```suggestion
`jsonb_path_ops` especially should use hash partitioning since the index key is itself a hash, but hash partitioning GIN indexes is currently unsupported.
```